src/EnergyPlus/Psychrometrics.cc

```diff
@@ -728,6 +730,11 @@ namespace Psychrometrics {
         // Compared to Table 3 values (August 2007) with average error of 0.00%, max .30%,
         // min -.39%.  (Spreadsheet available on request - Lawrie).
 
+        // Note: the ASHRAE Handbook of Fundamentals  is being slightly inaccurate in its wording,
+        // and referring to Eq 5 applying to -100°C to 0°C and Eq 6 applying to 0°C to 200°C
+        // In fact, it is **not** 0°C, but the triple-point of water, which is 0.01°C. Evaluating the Eq 5 and 6 up to and from the triple-point
+        // is removing the discontinuity altogether.
```

Comment:
New note

Reply:
I think you were actually the one that found this critical point. Nice!